src/main/fulcro/websockets.clj

```diff
@@ -132,21 +138,21 @@
 
   If you don't supply a server adapter, it defaults to http-kit.
   "
-  ([parser]
-   (make-websockets parser nil {}))
-  ([parser http-server-adapter sente-socket-server-options]
-   (map->Websockets {:server-options (merge {:user-id-fn (fn [r] (:client-id r))} sente-socket-server-options)
-                     :server-adapter (or http-server-adapter (hk/get-sch-adapter))
-                     :parser         parser})))
+  [parser websockets-uri {:keys [http-server-adapter transit-handlers sente-options]}]
```

Comment:
Seems like moving these all into a map of options might be better. Especially for optional things...so perhaps the signature should be `(fn [parser options])`?

Reply:
That was my intention too, but the `websockets-uri` is not optional. At least not right now, on both the client and server. Do you want me to make them optional on both, with default value of `"/chsk"`?